src/verifiers/issuerIdentity/index.ts

```diff
@@ -0,0 +1,27 @@
+import { issuerIdentityVerifierBuilder } from "./builder";
+import {
+  OpenAttestationDidIdentityProof,
+  OpenAttestationDnsDidIdentityProof,
+  OpenAttestationDnsTxtIdentityProof,
+} from "./verifiers";
+
+// Default set of issuer identity verifiers
+// Currently excludes OpenAttestationDidIdentityProof
+export const defaultIssuerIdentityProofVerifiers = [
+  OpenAttestationDnsDidIdentityProof,
+  OpenAttestationDnsTxtIdentityProof,
+];
+
+// Default ISSUER_IDENTITY verifier, built from defaultIssuerIdentityProofVerifiers
+export const OpenAttestationIssuerIdentityVerifier = issuerIdentityVerifierBuilder(defaultIssuerIdentityProofVerifiers);
```

Comment:
this bring a difference in how we treat the different kind of verifiers. For instance now we also cant mix different kind of document status (for instance one issuer using document store and one using token registry)